test_data/generated_test_data/ontologyR2RV2/knoraApiOntologyWithValueObjects.jsonld

```diff
@@ -8576,7 +8576,7 @@
             "@id": "knora-api:isAnnotationOfValue"
         },
         {
-            "rdfs:label": "is audio segment of",
+            "rdfs:label": "Segment of (Audio)",
```

Comment:
What's the reason that type of the segment is in brackets?

Reply:
Thanks for the explanation. Yes, sorry, I didn't mention it in the previous comment. W/o brackets it looks for me more naturally. I would even say that the brackets actually highlight the "Audio" part even more ;)

Reply:
I removed that part entirely - it should always be clear what type the target has anyway, so the information is redundant.